client/my-sites/themes/theme-activation-confirmation-modal.jsx

```diff
@@ -154,46 +182,49 @@ class AutoLoadingHomepageModal extends Component {
 					},
 					{
 						action: 'activeTheme',
-						label: translate( 'Activate %(themeName)s', { args: { themeName } } ),
+						label: translate( 'Activate %(themeName)s', { args: { themeName: theme.name } } ),
```

Comment:
I think you can simplify the args like you have with the others:

```args: { themeName }```

Reply:
Not in this case, I'm afraid. If the theme is retired, `themeName` is the name of the active theme. So in reality, you'd be asking a user if they want to activate their current theme.

That being said, I completely agree that this is a little confusing so I've switched `theme` to `installingTheme`.